src/fastcs_odin/frame_processor.py

```diff
@@ -78,6 +80,7 @@ class FrameProcessorPluginController(OdinAdapterController):
     """SubController for a plugin in a frameProcessor application."""
 
     async def initialise(self):
+        await self._create_commands()
         if any("dataset" in p.path for p in self.parameters):
```

Comment:
Not really part of this change, but could you pull the whole if block out into a `_create_dataset_controllers` function, just so that the `create_commands` call isn't lost in the noise?